api/integrations/github/views.py

```diff
@@ -71,6 +71,23 @@ def create(self, request, *args, **kwargs):
             if re.search(r"Key \(organisation_id\)=\(\d+\) already exists", str(e)):
                 raise DuplicateGitHubIntegration
 
+    def destroy(self, request, *args, **kwargs):
+        instance = self.get_object()
+        installation_id = instance.installation_id
+        token = generate_jwt_token(settings.GITHUB_APP_ID)
+
+        url = f"{GITHUB_API_URL}app/installations/{installation_id}"
+
+        headers = {
+            "X-GitHub-Api-Version": GITHUB_API_VERSION,
+            "Accept": "application/vnd.github.v3+json",
+            "Authorization": f"Bearer {token}",
+        }
+
+        response = requests.delete(url, headers=headers, timeout=10)
+        response.raise_for_status()
```

Comment:
We're assuming we're returning 500 in case of any errors from Github here?

Do we want to return the error message to the client or at least log it in some way?

Reply:
Right, good catch. Done.